scripts/regtest.sh

```
@@ -121,12 +128,42 @@ cmd_test() {
+  if [ "$PKG" != "ts-sdk" ]; then
+    echo "groups: only defined for ts-sdk" >&2
+    exit 1
+  fi
+  local failed=()
+  while IFS=$'\t' read -r name files; do
+    [ -n "$name" ] || continue
+    echo "=== e2e group: $name ==="
+    cmd_reset
+    cmd_up
+    cmd_setup
+    # shellcheck disable=SC2086 -- $files is a deliberate word-split file list
```

Shellcheck (0.11.0)

[error] Couldn't parse this shellcheck directive. Fix to allow more checks.

(SC1073)